docs/development/testing_guide.rst

```diff
@@ -1,50 +1,49 @@
 .. _testing-guidelines:
 
-******************
-Testing Guidelines
-******************
-
-.. _testing-guidelines-motivation:
+*************
+Writing Tests
+*************
 
-Motivation
-==========
+Code contributions to PlasmaPy that create or change functionality must
+include tests and documentation before being merged.
 
-Tests are vital for software reliability and maintainability.  Writing
-tests requires additional effort now, but saves considerable time in the
-long run.  Tests enable us to modify code and quickly discover when we
-introduce errors [1]_.  Tests also provide future contributors with
-examples of how functions and classes were originally intended to be
-used.
+Tests are vital for software reliability and maintainability. Writing
+tests requires additional effort in the short term, but saves
+considerable time in the long term. Tests enable us to modify code and
+discover bugs as soon as we introduce them [1]_. When a new bug is
+discovered, adding a corresponding test greatly reduces the likelihood
+of that bug showing up in the future [2]_.
 
-Tests should be readable and maintainable.  Well-written tests are
+Tests should be readable and maintainable. Well-written tests are
 easier to understand and modify when the behavior of a function or
-method is changed.  Consequently, tests should be held to the same
+method is changed. Consequently, tests should be held to the same
 code quality standards as the rest of the package.
 
-When bugs are discovered, they should be turned into test cases to
-prevent the bug from emerging again in the future [2]_.
+.. tip::
+
+   Writing tests takes time, but debugging takes more time.
```

Comment:
Thanks to @StanczakDominik for this quote from the software testing tutorial during Plasma Hack Week!

Reply:
... I spent most of yesterday debugging tests. Should we mention that?